src/ansible_navigator/configuration_subsystem/configurator.py

```diff
@@ -17,6 +17,8 @@
 from ..utils import oxfordcomma
 from .definitions import ApplicationConfiguration
 from .definitions import Constants as C
+
+# pylint: disable=deprecated-module
```

Comment:
Why is it deprecated in the first place?

Reply:
I do not know but @cidrblock is investigating.

Reply:
Sorted by switching to official pylint repo and last stable release.

Reply:
I don't see this in the current PR, if I recall this was a bug in pylint related to the deprecation of `Parser` in 3.9 but was fixed in a later pylint release  (we have a `Parser` too, unique to navigator)